composer.json

```diff
@@ -4,15 +4,18 @@
     "minimum-stability": "dev",
```

Comment:
Please try to use same composer.json, discass in team the version of php you all are using.